### ISSUE TITLE
Component testing - error `Performance.mark: Global object is unavailable`

### ISSUE THREAD

**Issue by gsouf** [cypress-io/cypress#23282]
### Current behavior

After modifying cypress config ; When reloading a component test (ie. making a change in a component or in spec file and saving the file) then the error `Performance.mark: Global object is unavailable` will appear in the spec window:

![image](https://user-images.githubusercontent.com/3215399/184143577-a05eae30-ddd3-4d37-8342-440e9b7a046e.png)

Reloading the page will fix the issue.

### Desired behavior

No error

### Test code to reproduce

- Clone repo https://gitlab.com/souf/react-webpack-playground/-/tree/cypress-reload-error using branch `cypress-reload-error`
- run `yarn install` and `yarn cypress:open`.
- open component testing for `App`.
- change something in the cypress config in `/cypress.config.js` for example the value of `defaultCommandTimeout`.
- open file /src/components/App.jsx change the "Hello" string and save to trigger a reload.

Error message should occur.

### Cypress Version

10.4.0

### Other

Using react+webpack setup. Only tested on firefox linux

**Comment by AtofStryker** [comment]
Hi @gsouf.Thank you for opening an issue. I see you cant find a way to reproduce this. Are you able to provide some additional details, or a reproduction repository that at least intermittently reproduces the issue? Does it only happen in firefox?

**Comment by gsouf** [comment]
@AtofStryker a bit after I opened it was occuring **everytime** i saved a file. I just re-opened my dev environment and it's not occuring anymore. Let me try again tomorrow when I'll be actively working on it.

In interim do you have any idea what is trying to invoke `Performance.mark`? Is that from cypress or could that be from react?

**Comment by gsouf** [comment]
@AtofStryker I found a way to reproduce the error. You'll find the description updated. It occurs after modifying the cypress config once. I couldn't test on other browsers because for some reaosns I cant manage to get cypress to find my other browsers on the machine. I tested only on linux.

I think the error is specific to firefox though. I found this recent where they added the error message `Global object in unavailable` https://hg.mozilla.org/mozilla-central/rev/139c89a60b7261a619deb3cb40a997cc6a295ec0 and this ticket where it is mentioned https://bugzilla.mozilla.org/show_bug.cgi?id=1775499

**Comment by gsouf** [comment]
@AtofStryker I also noticed it occurs after modifying a sass file that is imported in my component. It is setup with `style-loader` / `style-loader` / `sass-loader`.

I'll assume this is the same issue as when we modify the config so I don't think it's worth it providing a reproducing example for that

**Comment by AtofStryker** [comment]
@gsouf I gave your repository a clone and tried the reproduction steps. I edited the config, but didn't get the error you are seeing. I am also on MacOS though so that could be different. Does it happen every time with those steps?

![Screen Shot 2022-08-15 at 5 36 02 PM](https://user-images.githubusercontent.com/3980464/184722998-97acc8ee-461c-48be-b927-a5839a84e470.png)

**Comment by gsouf** [comment]
@AtofStryker yes I confirm it happens everytime.

- Modify the config 
- save
- it works
- modify the component file
- save
- it breaks

I was also able to setup chrome today and I confirm the issue also exists in chrome (google chrome stable downloaded today). But the issue is slightly different. There is no error message but no success message either (see screenshot bellow), nothing in the dev tools console either and at this point the chrome is frozen. You can see the "run all specs button" is still in the "square icon" status. Reloading the page or trying to close browser does not work.

All I can do at this point is to close the cypress process to get chrome to close.

![image](https://user-images.githubusercontent.com/3215399/184842550-fb019b07-6f76-40a8-86dd-0544a92f7153.png)

**Comment by rachelruderman** [comment]
Hi @gsouf, thank you for the detailed steps to reproduce and especially for the clonable repo. I was able to confirm the behavior you reported on both Firefox 101 and Chrome 104. We will now route this to the appropriate team to fix 🎉 🙏
